.github/workflows/rust.yml

```diff
@@ -342,3 +342,67 @@ jobs:
             target/wix/pixi-${{ matrix.target }}.msi
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  test-examples:
```

Comment:
Im wondering if it would make sense to put this in its own workflow.. Dunno if thats easy to do..

Reply:
Yes I tried that first but the artifact sharing would be very hacky, with url's and such.